include/titan/options.h

```diff
@@ -172,6 +172,7 @@ struct TitanCFOptions : public ColumnFamilyOptions {
   // data's 0s and 0s created by punch holes).
   uint64_t block_size{4096};
   bool enable_punch_hole_gc{false};
+  uint64_t punch_hole_threshold{4 * 1024 * 1024};
```

Comment:
Can we let `punch_hole_threshold` == 0 indicate punch hole gc disabled

Reply:
Done